scripts/make_msd_index.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -eux
```

Comment:
Can you list the dependencies and make the folder name a variable so that it would be easy for someone to rerun this?